timer.go

```diff
@@ -0,0 +1,51 @@
+package quic
```

Comment:
I wouldn't have two almost identical structs that subtly differ in their behavior in two separate packages with a similar name. Maybe call this `connectionTimer` or something like that?

Alternatively, you could generalize this helper by making it non-specific to the connection?

Reply:
Done.